app/bundles/ReportBundle/Views/FormTheme/Report/aggregator_row.html.twig

```diff
@@ -0,0 +1,13 @@
+{% set labelClass = form.vars.label_attr.class is not empty ?: 'control-label' %}
+{% block aggregator_row %}
```

Comment:
Custom form themes should have `_` in front of the block name. ie `_aggregator_row` or it will never be used.